geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/list/AbstractLTrimIntegrationTest.java

```diff
@@ -147,6 +147,31 @@ public void removesKey_whenRangeIsEmpty(long start, long end) {
     };
   }
 
+  @Test
+  @Parameters(method = "getRangesForOneElementList")
+  @TestCaseName("{method}: start:{0}, end:{1}, expected:{2}")
+  public void trimsToSpecifiedRange_givenListWithOneElement(long start, long end,
+      String[] expected) {
+    jedis.lpush(KEY, "e1");
+
+    jedis.ltrim(KEY, start, end);
+    assertThat(jedis.lrange(KEY, 0, -1)).containsExactly(expected);
+  }
+
+  @SuppressWarnings("unused")
+  private Object[] getRangesForOneElementList() {
+    // Values are start, end, expected
+    // For initial list of {e4, e3, e2, e1}
```

Comment:
This comment is incorrect

Reply:
Yanked, since it's pretty simple for a one-element list.